### ISSUE TITLE
[Q] What determines whether a chart goes in Custom Charts or Hidden panels?

### ISSUE THREAD

**Issue by richardrl** [wandb/wandb#3639]
I used the same code to generate two line plots:

```
wandb_lol, wandb_keys = wandb_process_out_dic(process_state_dict_arr_with_vector_values(parameter_model_state_dict_arr, num_internal_bodies))

        # TODO: currently doesnt work
        # float_dic = process_single_state_dict(pm.get_summary_dict(), num_internal_bodies)
        # for k, v in float_dic.items():
        #     table.add_data(training_iteration, k, v)

        table = wandb.Table(data=make_data(wandb_lol, wandb_keys), columns=["step", "lineKey", "lineVal"])

        line_chart = wandb.plot_table(vega_spec_name="richardrl/color-dash-swap",
                         data_table=table,
                         fields= {"step": "step", "lineKey": "lineKey", "lineVal": "lineVal"},
                         string_fields={"title": "Parameter Model Summary", "xname": "Alternation Iterations"}
                         )

        wandb.log({"parameter_model_summary": line_chart})


        wandb_lol, wandb_keys = wandb_process_out_dic(process_state_dict_arr_with_scalar_values(cdda_master))
        table = wandb.Table(data=make_data(wandb_lol, wandb_keys), columns=["step", "lineKey", "lineVal"])

        line_chart = wandb.plot_table(vega_spec_name="richardrl/color-dash-swap",
                         data_table=table,
                         fields= {"step": "step", "lineKey": "lineKey", "lineVal": "lineVal"},
                         string_fields={"title": "Cost Summary", "xname": "Alternation Iterations"}
                         )

        wandb.log({"cost_summary": line_chart})
```

However, the "Cost Summary" is in Custom Charts, whereas the "Parameter Model Summary" is in Hidden Panels. Why? How do we control this?

**Comment by armanhar** [comment]
Hey Richard, you can specify the section by adding a prefix like this:

`wandb.log({"section-a/metric": idx})
`

**Comment by armanhar** [comment]
Hey @richardrl, ​I wanted to follow up on this request. Please let us know if we can be of further assistance or if your issue has been resolved.
​

**Comment by richardrl** [comment]
@armanhar Thanks, I will try that.

**Comment by sieumap43** [comment]
> Hey Richard, you can specify the section by adding a prefix like this:
> 
> `wandb.log({"section-a/metric": idx}) `

What about `wandb.summary` ? How to show summary by default and not manually adding panels for each of the metrics summary? `wandb_run.summary["section/metric"] = value` does not appear in a default panel like `wandb.log({"section-a/metric": idx}) `

**Comment by paulacanva** [comment]
Even using `section` my chart ends in the hidden panel:

```
f"{self.WANDB_SECTION}Confusion Matrix:": wandb.sklearn.plot_confusion_matrix(
                    y_true, y_pred
                ),
```

I train two models and create confusion matrices for both. The sections for them are different, but they are both put in the same panel and it becomes a mess with duplicated labels.
<img width="512" alt="Screenshot 2023-05-26 at 10 58 10 am" src="https://github.com/wandb/wandb/assets/113396119/b2eb63fe-5e10-4b37-8f7d-fa7f83c9519c">
